shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/SurveyQuestionViewModel.kt

```diff
@@ -3,44 +3,35 @@ package co.nimblehq.avishek.phong.kmmic.presentation.module
 import co.nimblehq.avishek.phong.kmmic.domain.model.Survey
 import co.nimblehq.avishek.phong.kmmic.domain.usecase.SubmitSurveyAnswerUseCase
 import co.nimblehq.avishek.phong.kmmic.helper.DispatchersProvider
-import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.QuestionUiModel
-import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.toQuestionUiModel
-import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.toSurveySubmission
-import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.flow.catch
-import kotlinx.coroutines.flow.flowOn
-import kotlinx.coroutines.flow.launchIn
-import kotlinx.coroutines.flow.onEach
-import kotlinx.coroutines.flow.onStart
-import kotlinx.coroutines.flow.update
+import co.nimblehq.avishek.phong.kmmic.presentation.uimodel.*
+import kotlinx.coroutines.flow.*
 
 data class SurveyQuestionViewState(
     val isLoading: Boolean,
     val isSuccess: Boolean = false,
     val errorMessage: String? = null,
     var questions: List<QuestionUiModel> = listOf(),
-    val backgroundImageUrl: String = ""
+    val backgroundImageUrl: String = "",
 ) {
     constructor() : this(isLoading = false)
 }
 
 class SurveyQuestionViewModel(
     private val submitSurveyAnswerUseCase: SubmitSurveyAnswerUseCase,
-    private val dispatchersProvider: DispatchersProvider
+    private val dispatchersProvider: DispatchersProvider,
 ) : BaseViewModel() {
 
     private val _viewState: MutableStateFlow<SurveyQuestionViewState> =
         MutableStateFlow(SurveyQuestionViewState())
-    val viewSate: StateFlow<SurveyQuestionViewState> = _viewState
+    val viewState: StateFlow<SurveyQuestionViewState> = _viewState
```

Comment:
Would you help me to fix the issue related to my typo? The fix is replacing `SurveyQuestionCombineViewModel:25` in iOS module with this `createPublisher(for: sharedViewModel.viewStateNative)` 🙏

Reply:
@phongvhd93 done https://github.com/nimblehq/avishek-phong-kmm-ic/pull/139/commits/34aa0f91ac2cecbe7b0c17ba05c2703414abd7af